lms/static/sass/features/_learner-profile.scss

```diff
@@ -265,14 +271,15 @@
       margin: 0 auto;
       border-bottom: 1px solid $gray-l3;
       background-color: $gray-l4;
-      padding: ($baseline*0.75) 0;
+      padding: ($baseline*0.75) 5%;
```

Comment:
Nit: I don't love percentage-based padding, but maybe there's a good reason for it.